src/platformdirs/macos.py

```diff
@@ -1,9 +1,20 @@
 import os
 
 from .api import PlatformDirsABC
+from .unix import Unix
 
+XDG_SUPPORT = {
+    "user_data_dir": "XDG_DATA_HOME",
+    # "site_data_dir": "XDG_CONFIG_DIRS",  # Not supported
```

Comment:
Why not supported?

Reply:
Decided against, because a quick survey of what I have installed on my Mac shows that nothing actually used the `site_*`/`*_DIRS` variables except in vendored copies of appdirs. Actual usage of this seems to be much rarer than usage of `XDG_{CONFIG,DATA,CACHE}_HOME`, which I believe is the part with the most adoption in practice.

As for logging: macOS's Console.app reads from the well-known platform logging directories. As such, it's a violation of platform expectations to drop logs elsewhere.

Reply:
> Decided against, because a quick survey of what I have installed on my Mac shows that nothing actually used the `site_*`/`*_DIRS` variables except in vendored copies of appdirs. Actual usage of this seems to be much rarer than usage of `XDG_{CONFIG,DATA,CACHE}_HOME`, which I believe is the part with the most adoption in practice.

I'd rather follow the XDG spec then such local quick surveys, unless macOS clearly advises against it (as is the case of the logging).